Automattic/jetpack
# Jetpack, Jetpack Social: Update icon to JetpackLogo in pre-publish panels

Part of #26048

Addresses [comment](https://github.com/Automattic/jetpack/issues/26048#issuecomment-1239471291)
> It'd be good to make them slightly smaller, and try them dark rather than grey.



#### Changes proposed in this Pull Request:

* Resize and updates Jetpack color. Took `#444444` from Jetpack's style in Figma

### After

**On Jetpack**
![image](https://user-images.githubusercontent.com/746152/189012849-a0c87c4e-4762-46fd-9bee-cc9fa3341bbd.png)


**On Jetpack Social**
![image](https://user-images.githubusercontent.com/746152/189013350-5d508a4d-386b-45d0-bd78-3cd0b044fb67.png)
### Before

**On Jetpack**

![image](https://user-images.githubusercontent.com/746152/189013070-69563eda-bbb9-4a37-8bfc-cfe0871067dc.png)


**On Jetpack Social**

![image](https://user-images.githubusercontent.com/746152/189012965-c3cb9592-d67e-4faa-99f2-1aa7d0218e04.png)


#### Other information:


#### Jetpack product discussion
p1HpG7-hDK-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* On a connected site
* Enable Subscriptions
* Enable SEO tools
* Enable Social tools
* Start writing a post
* Hit "Publish" and stop before the next confirmation button. Confirm in the right sidebar that you see the Jetpack logo on each of the following sections, smaller and with a darker color (see screenshots):
  * Social Previews Panel
  * SEO Description Panel
  * Subscribers Panel
  * Share this Post Panel
